### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by Robert-G-J**
This:
- Adds comment model
- Associates comments with posts
- Adds comments controller
- Renders views
🥇 @sophieklm  🥇

### CODE REVIEW

**Comment by Robert-G-J** [comment]
I think the Travis build _might_ be failing because we haven't yet a travis.yml file, so Travis' build doesn't like the Ruby version (as a minimum). Ideas?

**Comment by sophieklm** [comment]
[Posts can have comments](https://trello.com/c/DGblN31C/16-posts-can-have-comments)

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  def create
+      @post = Post.find(params[:post_id])
+      @comment = @post.comments.create(comment_params)
+      redirect_to posts_path
+    end
+
+    private
+      def comment_params
+        params.require(:comment).permit(:commenter, :body)
+      end
```
Comment (dbugsy) [inline app/controllers/comments_controller.rb]: fix indentation

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
-  resources :posts
+  resources :posts do
+    resources :comments
+  end
```
Comment (dbugsy) [inline config/routes.rb]: nice resource nesting

**Review by dbugsy — APPROVED** [review]
Fix your indendation, then it's good to merge.
:shipit: :shipit:

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (sophieklm) [reply app/controllers/comments_controller.rb]: updated

**Review by AlexChalk — APPROVED** [review]
:shipit:

**Review by anaalta — APPROVED** [review]
:shipit:  Clean and easy to read!

**Comment by sophieklm** [comment]
Had to update a thing, is it still good? @ddosque

**Comment by SimonTanner** [comment]
:shipit:

**Pull request merged**